data/json/items/comestibles/brewing.json

```diff
@@ -554,7 +554,7 @@
     "id": "brew_moonshine",
     "looks_like": "moonshine",
     "name": { "str_sp": "moonshine mash" },
-    "description": "Unfermented moonshine.  Just some water, sugar and corn, like good ol' auntie's recipe.  Using pumpkin instead of corn would reduce the amount of extra sugar you need to add.",
+    "description": "Unfermented moonshine.  Just some water, sugar and corn, like good ol' auntie's recipe.  Using pumpkin instead of corn would reduce the amount of extra sugar you need to add.  You need to put it in fermenting vat to brew it.",
```

Comment:
```suggestion
    "description": "Unfermented moonshine.  Just some water, sugar and corn, like good ol' auntie's recipe.  Using pumpkin instead of corn would reduce the amount of extra sugar you need to add.  You need to put it in a fermenting vat to brew it.",
```